spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/EnableBinding.java

```diff
@@ -31,7 +31,7 @@
 import org.springframework.integration.config.EnableIntegration;
 
 /**
- * Enables the binding of inputs and outputs to a broker, according to the list
+ * Enables the binding of {@link Input} and {@link Output} message channels to a broker, according to the list
```

Comment:
ok, that was my first impression too. I think `@Input` and `@Output` annotations are always used to denote the message channels in our implementation. Hence I added the same here as well.